Doc/library/venv.rst

```diff
@@ -61,7 +61,130 @@ See :pep:`405` for more background on Python virtual environments.
 Creating virtual environments
 -----------------------------
 
-.. include:: /using/venv-create.inc
+:ref:`Virtual environments <venv-def>` are created by executing the ``venv``
+module:
+
+.. code-block:: shell
+
+    python -m venv /path/to/new/virtual/environment
+
+This creates the target directory (and any parent
+directories that don't already exist) and places a ``pyvenv.cfg`` file in it
+with a ``home`` key pointing to the Python installation from which the command
+was run (a common name for the target directory is ``.venv``).  It also creates
+a ``bin`` (or ``Scripts`` on Windows) subdirectory containing a copy/symlink
+of the Python binary/binaries (as appropriate for the platform or arguments
+used at environment creation time). It also creates an (initially empty)
+``lib/pythonX.Y/site-packages`` subdirectory (on Windows, this is
+``Lib\site-packages``). If an existing directory is specified, it will be
+re-used.
+
+.. versionchanged:: 3.5
+   The use of ``venv`` is now recommended for creating virtual environments.
+
+.. deprecated-removed:: 3.6 3.8
+   :program:`pyvenv` was the recommended tool for creating virtual environments
+   for Python 3.3 and 3.4, and replaced in 3.5 by executing ``venv`` directly.
+
+.. highlight:: none
+
+On Windows, invoke the ``venv`` command as follows::
+
+    c:\>Python35\python -m venv c:\path\to\myenv
+
+Alternatively, if you configured the ``PATH`` and ``PATHEXT`` variables for
+your :ref:`Python installation <using-on-windows>`::
+
+    c:\>python -m venv c:\path\to\myenv
```

Comment:
We should present the most common scenario (which isn't Python 3.5 built from source)
```suggestion
On Windows, invoke the ``venv`` command as follows:

.. code-block:: ps1con

   PS> python -m venv C:\path\to\new\virtual\environment
```